src/dfx/src/commands/nns/import.rs

```diff
@@ -24,7 +24,7 @@ pub async fn exec(env: &dyn Environment, opts: ImportOpts) -> DfxResult {
     let mut config = config.as_ref().clone();
 
     let network_mappings = get_network_mappings(&opts.network_mapping)?;
-    let ic_commit = replica_rev();
+    let ic_commit = std::env::var("DFX_IC_REF").unwrap_or_else(|_| replica_rev().to_string());
```

Comment:
"ic ref" already has a meaning in dfx, which is the IC reference implementation.   Could we call this `DFX_IC_COMMIT` instead?

Reply:
Sure thing.  Done.